spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "spec_helper"
+
+require "rails_helper"
+
+RSpec.describe("Friendships", type: :feature) do
+  let(:user2) { User.create(name: "user", email: "user@gmail.com", password: "123456") }
+
+  scenario "User approving or rejecting friend requests status" do
+    visit "/users/sign_in"
+
+    enter "Email", with: user2.email
+    enter "Password", with: user2.password
+    click_button "Log in"
+    expect(find(".notice")).to(have_content("Signed in successfully."))
+    visit "/users"
+    expect(page).to(have_text("User profile"))
+  end
```

Comment:
Sorry but I don't see where rejecting or approving is in this scenario?
The scenario ends at the user profile page. and what should we understand, please?